addon/components/standard-stat.js

```diff
@@ -1,7 +1,38 @@
 import Ember from 'ember';
 
-const { Component } = Ember;
+const {
+  Component,
+  computed
+} = Ember;
 
 export default Component.extend({
   classNames: ['upf-stat'],
+
+  defaultRadialSize: 'upf-progress-radial-lg',
+  defaultRadialClass: 'upf-progress-radial--none',
+
+  type: null,
+  size: null,
```

Comment:
I think instead of use `defaultRadialSize` why not just set `size: 'upf-progress-radial-lg'` ? Same for `type`